ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -96,6 +96,7 @@ import SwiftUI
     var primaryText: String? { get set }
     var secondaryText: String? { get set }
     var ringColor: UIColor? { get set }
+    var imageBasedRingColor: UIImage? { get set }
```

Comment:
do we really need the image based ring? I was wondering similar to Mac if UIColor (uiimage backed) is enough for avatar rainbow ring.

Reply:
I tried using a UIColor creates with an image pattern (https://developer.apple.com/documentation/uikit/uicolor/1621933-init) but it does not seem to work with the SwiftUI Color. It just does not render anything.

I also found a report of the same thing in a stack overflow thread:
- https://stackoverflow.com/questions/60749545/swiftui-cant-create-color-by-uicolorpatternimage)

Alternatively we could create a gradient in SwiftUI, but that would make the rainbow ring a very specific property and potentially not match other controls that could be using an image to create the pattern.

Reply:
oh wow. that's suprising to know. maybe we should add note why we are doing thing in comment then?

Reply:
Sounds like an Apple Bug?

Reply:
I think it's unlikely that Apple will consider this as a bug since they provide the ImagePaint to accomplish that job.